src/resources/js/components/layout/SearchBar.vue

```diff
@@ -2,13 +2,56 @@
     <v-autocomplete
         single-line
         hide-details
-        placeholder="Start typing to Search"
+        placeholder="Rechecher une presentation"
         append-icon="mdi-magnify"
+        hide-no-data
+        v-model="select"
+        :loading="loading"
+        :items="items"
+        item-value="id"
+        item-text="title"
+        :search-input.sync="search"
+        @change="GoToPresentation(select)"
     ></v-autocomplete>
 </template>
 
 <script>
 export default {
   name: 'SearchBar',
+  data() {
+    return {
+      loading: false,
+      items: [],
+      search: null,
+      select: null,
+    };
+  },
+  watch: {
+    search(val) {
+      return val && val !== this.select && this.querySelections(val);
+    },
+  },
+  methods: {
+    querySelections(v) {
```

Comment:
use more specific name instead of `v` i have to read the code to understand what is the purpose of this var

Reply:
ok rename in [8c56a](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/152/commits/8c56ae323d95a4e203eaf6cc54ea5212b14fe42c)